sections/introduction.tex

```diff
@@ -1,8 +1,13 @@
 \section{Introduction}\label{sec:introduction}
+% Intro to meta data
+Data is everywhere and its used in many different scenarios and use cases. 
+When referring to a data set, we talk about a data having meta-data information or attributes, which is the entries that could describe certain features about the data, like petal length from well known IRIS data set\footnote{https://archive.ics.uci.edu/ml/datasets/iris} describing the length of a particular flowers petals.
```

Comment:
Er ikke sikker på at jeg er enig med denne måde at beskrive metadata på. Ville mene at petal length i IRIS er 'normal' ikke meta-data, da det virker som den centrale information i det dataset. IDK :/

Reply:
```suggestion
When referring to a data set, we talk about a dataset having meta-data information, which are non-essential data features. One example of this could be publication date of a text document. In most contexts it is not needed to use and understand the document, however it provides more context to the main content of the the document: the text.
```
Her er et forsøg på at stadig givc et eksempel, men kunne ikke rigtig tænke på et konkret kendt dataset.